src/devices/LidarLiteV3/SystemStatusFlag.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.TimeOfFlight.Models.LidarLiteV3
+{
+    /// <summary>
+    /// System status flags
+    /// </summary>
+    [Flags]
+    public enum SystemStatusFlag
+    {
+        /// <summary>
+        /// Process Error Flag
+        /// 0 - No error detected
```

Comment:
if possible remove 0s and 1s from the summary

Reply:
fixed